README.md

```diff
@@ -1,2 +1,121 @@
-# HelloMicroverse
-First Microverse Project
+<a name="readme-top"></a>
+
+<div align="center">
+
+<h3><b>First Project: Hello Microverse</b></h3>
+<br>
+<h4><b>Alejandro Orozco</b></h4>
+
+</div>
+<br>
+<br>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> This is an initial project, using HTML & CSS ...
+
+## 🛠 Built With <a name="built-with"></a>
+
+1. HTML
+2. CSS
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+<br>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added HTML files
+- Added CSS files
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Coming soon...
```

Comment:
- [ ] Since you don't have the live link present in the project, I'll kindly suggest you take that section out.
- This makes your README concise with information only needed as regards the project 👍